components/_patterns/01-atoms/02-text/00-headings/_heading.twig

```diff
@@ -18,7 +18,12 @@
 #}
 {% set heading_base_class = heading_base_class|default('h' ~ heading_level) %}
 
-<h{{ heading_level }} {{ bem(heading_base_class, (heading_modifiers), heading_blockname) }}>
+{% set additional_attributes = {
+  "class": [bem(heading_base_class, (heading_modifiers), heading_blockname)],
```

Comment:
This is great! works for me! But I think it definitely needs to be in documentation, not actually implemented here by default.